vic/drivers/image/src/vic_force.c

```diff
@@ -334,13 +334,12 @@ vic_force(void)
             atmos[i].air_temp[j] -= CONST_TKFRZ;
             // precipitation in mm/period
             atmos[i].prec[j] *= global_param.snow_dt;
-            // pressure in kPa
-            atmos[i].pressure[j] /= PA_PER_KPA;
```

Comment:
@yixinmao: So the units of air pressure (input and output) should be Pa? Good to know! This was still not clear for me :).

But still: with this fix the output is still a factor 1000 smaller than input/forcing (without the fix it is even a factor 1.000.000 smaller).